File: Lavida Online Judge/#1091 이차 방정식의 해/이차 방정식의 해.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int test, a, b, c;
        test = sc.nextInt();
        for(int i = 0; i<test; i++){
            a = sc.nextInt();
            b = sc.nextInt();
            c = sc.nextInt();
            if((b*b - 4 *a*c)>0){
                System.out.println("This Equation has two answers");
                }
            else if((b*b - 4 *a*c)==0){
                System.out.println("This Equation has only one answer");
            }
            else if((b*b - 4 *a*c)<0){
                System.out.println("This Equation has no answer");
            }
        }
    }
}

File: Lavida Online Judge/#1091 이차 방정식의 해/이차 방정식의 해.c
#include <stdio.h>

int main() {
	int test, a, b, c, k;

	scanf("%d", &test);

	while (test--) {
		scanf("%d %d %d", &a, &b, &c);
		k = (b * b) - 4 * a * c;

		if (k > 0)
			printf("This Equation has two answers\n");
		else if (k == 0)
			printf("This Equation has only one answer\n");
		else if (k < 0)
			printf("This Equation has no answer\n");
	}
	return 0;
}